roboteam_ai/src/utilities/Constants.h

```diff
@@ -11,62 +11,67 @@ namespace rtt {
 namespace ai {
 
 typedef std::tuple<double, double, double> pidVals;
+typedef std::tuple<double, double, double, double> pidfVals;
 
 class Constants {
 
     public:
         static void init();
         static bool GRSIM();
 
+        /// TICK RATE ///
+        static constexpr int GAME_ANALYSIS_TICK_RATE()      { return 30; };
+        static constexpr int TICK_RATE()                    { return 60; };
+
         /// LOGGING ///
         static bool SHOW_LONGEST_TICK();
         static bool SHOW_TICK_TIME_TAKEN();
 
-        static bool SHOW_NUMTREE_TIME_TAKEN();
         static bool SHOW_COACH_TIME_TAKEN();
 
+        static bool SHOW_NUMTREE_TIME_TAKEN();
         static bool SHOW_NUMTREE_DEBUG_INFO();
         static bool SHOW_FULL_NUMTREE_DEBUG_INFO();
+
         static bool SHOW_BALL_HANDLE_DEBUG_INFO();
         static bool SHOW_FULL_BALL_HANDLE_DEBUG_INFO();
 
-        // Basic rulesets for rule compliance
         static std::vector<RuleSet> ruleSets();
 
-        /// ROBOT AND RELATED ///
+        /// ROBOT COMMANDS ///
         static double MAX_VEL_CMD();
         static int MAX_ID_CMD();
         static double MAX_ANGULAR_VEL_CMD();
         static double MIN_ANGLE();
         static double MAX_ANGLE();
+        static double MAX_ANGULAR_VELOCITY();       // Rad per second
 
-        // max velocities for refstates
-        static double MAX_VEL();
-        static double MAX_STOP_STATE_VEL();
-
-        static double MIN_VEL();  // Minimum velocity to make the robot move
+        /// ACCELERATION LIMITERS ///
+        static double MIN_VEL();        // Minimum velocity to make the robot move
         static double MAX_ACC_UPPER();  // Maximum acceleration for moving in the forward direction
         static double MAX_ACC_LOWER();  // Maximum acceleration for moving in the sideways direction
         static double MAX_DEC_UPPER();  // Maximum deceleration for moving in the forward direction
         static double MAX_DEC_LOWER();  // Maximum deceleration for moving in the sideways direction
 
-        static double MAX_VEL_BALLPLACEMENT();
-        static double MAX_ANGULAR_VELOCITY();    // Rad per second
-        static double ROBOT_RADIUS(); // TODO: Need to test if world_state agrees with this definition of the centre of the robot
-        static double ROBOT_RADIUS_MAX();
+        /// ROBOT SPECS ///
+        static constexpr double ROBOT_RADIUS()              { return 0.089; };
```

Comment:
this is bad for compilation time if we wanna change these constants, although i do see that we don't change them.